components/nested_star/example/main.cc

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <algorithm>
+#include <iomanip>
+#include <iostream>
+#include <string>
+#include <vector>
+#include "lib.rs.h"  // NOLINT
+
+int main(int argc, char** argv) {
+  uint8_t epoch = 1;
+  std::vector<std::string> input = {"TestMetricOne|1", "TestMetricTwo|2"};
+  std::string output;
+
+  auto public_key = nested_star::get_ppoprf_null_public_key();
+
+  auto rrs_res = nested_star::prepare_measurement(input, epoch);
```

Comment:
I know this is a bit painful, but we settled on the name "constellation" for the nested STAR encoding design, and we'll probably rename the rust repo to match. It's confusing having two names for things, so we should s/nested_star/constellation/ in this PR before it lands.

Reply:
renamed